parser/parser.go

```diff
@@ -153,6 +157,7 @@ func Parsers() []string {
 		VCL,
 		XML,
 		YAML,
+		SPDX,
```

Comment:
nit: Please continue ordering alphabetically as the others are.

Reply:
done